packages/blocks/src/api/utils.js

```diff
@@ -54,7 +54,7 @@ export function isUnmodifiedDefaultBlock( block ) {
 	const blockType = getBlockType( defaultBlockName );
 
 	return every(
-		blockType.attributes,
+		blockType?.attributes,
```

Comment:
I had to double check this in https://lodash.com/docs/#every, but it looks like lodash.every will return true on an empty/undefined collection.

It's probably more correct to return false at this point, but this case probably won't happen. If we unregister the default block, the default block name reverts to `null` and so we should exit on line 41.

<img width="505" alt="CleanShot 2021-09-16 at 15 45 28@2x" src="https://user-images.githubusercontent.com/1270189/133695087-7cd94ee1-bb09-49c0-b7e0-b1de353bbf2f.png">

Reply:
I'm removing these changes.. Since it is not going to happen. Indeed it will return at line 41. Apologize I didn't notice earlier.

Reply:
No worries, some of these are a bit tricky.